website/docs/docs/deploy/continuous-deployment.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Continuous deployment in dbt Cloud"
+sidebar_label: "Continuous deployment"
+description: "Learn about continuous deployment (CD) workflows "
+---
+
+To help you improve data transformations and to ensure up-to-date data products, you can run [merge jobs](/docs/deploy/merge-jobs) to implement a continuous deployment (CD) workflow in dbt Cloud. Merge jobs can automatically build modified models for you whenever a pull request (PR) merges, making sure the latest changes are in production (such as modifications to code that generates the tables). You don't need to wait for the next scheduled job to run. 
```

Comment:
When you say "such as modifications to code that generates the tables" this sounds like there are multiple examples of things that would rebuild the prod schema via a merge job. But isn't code changes the only example?

Reply:
this was in the source material i used to draft this page. good callout, updated the wording!